Bugzilla/Template.pm

```diff
@@ -807,7 +807,12 @@ sub create {
         my %units = ('KB' => 1024, 'MB' => 1024 * 1024, 'GB' => 1024 * 1024 * 1024,);
 
         if ($data < 1024) {
-          return "$data bytes";
+          if($data == 1) {
+            return "1 byte";
+          }
+          else {
+            return "$data bytes";
+          }
```

Comment:
This looks good, but it would be more idiomatic to use a ternary operator:
`return "$data byte" . ($data == 1 ? '' : 's');`